configure

```diff
@@ -39,16 +39,13 @@ sub_error()
     exit 1
 }
 
+# NOTE: Always quote your warning() messages
 warning()
```

Comment:
Add a comment explaining what parameters can/must be supplied (i.e. warning message and optional extra information)